test/change.test.js

```diff
@@ -610,71 +610,66 @@ describe('Change', function() {
 });
 
 describe('Change with with custom properties', function() {
-  var ChangeWithCustomProperty, TestModelWithModifiedChangeModel;
-  var customProperty = 'tenantId';
```

Comment:
Similarly here, I find the tests easier to read/understand when the property name is hard-coded.